tools/composer.json

```diff
@@ -7,5 +7,10 @@
     "phpstan/phpstan": "^1.0.0",
     "infection/infection": "^0.26.0",
     "friendsofphp/php-cs-fixer": "^3.1"
+  },
+  "config": {
+    "allow-plugins": {
+      "infection/extension-installer": true
+    }
```

Comment:
```suggestion
    "allow-plugins": false
```

Reply:
I don't think any composer plugins are needed

Reply:
I've opened this PR because tests are failing in #270 

![kép](https://user-images.githubusercontent.com/952007/178484768-f671813e-a591-4e47-a88f-21873bf98133.png)

Reply:
and that's fine, but tests are not failing due to plugins are not allowed but because explicit setting is missing